x-pack/plugins/cases/docs/openapi/components/schemas/customfields.yaml

```diff
@@ -0,0 +1,27 @@
+type: array
+description: Custom field values for a case.
+x-technical-preview: true
+# minItems: 0
+# maxItems: 5
```

Comment:
> # maxItems: 5

I have commented this out since I was able to add and change more than 5 fields in a single call.